spec/acceptance/ini_subsetting_spec.rb

```diff
@@ -145,6 +155,8 @@
   end
 
   describe 'show_diff parameter and logging:' do
+    basedir = setup_test_directory
+    setup_puppet_config_file
```

Comment:
Use `before` block.

Reply:
tried to add before block, but each time tests are failing. 
use it as it is, the config file will be created before, at the beginning of the execution of the tests and tests are passing.